controllers/spacebindingrequest/spacebindingrequest_controller.go

```diff
@@ -92,7 +99,360 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	}
 	logger.Info("spacebindingrequest found", "member cluster", memberClusterWithSpaceBindingRequest.Name)
 
-	// todo implement logic
+	if util.IsBeingDeleted(spaceBindingRequest) {
```

Comment:
It's not part of this PR but I missed the previous PR which introduced member cluster lookup. So I'm using this opportunity to point out that we can improve the lookup mechanism :)
Let's say there are two member clusters m-1 and m-2. And m-1 cluster is having an outage and temporally down. The reconcile won't work for all space requests even from m-2 if m-1 happens to be first in r.MemberClusters.

What we could do to mitigate it is too save the error in line 82 above but proceed with the lookup. And only if no SBR found in other members then return the saved error. Otherwise just log the error but do not return it. In that case m-2 won't be affected by the m-1 outage.

Feel free to address it in a separate PR though.

Reply:
Yeah, good point! The logic can be improved as you suggested, also I can check if it can be extracted to a reusable function since I see it's similar to what I'm using in spacerequest_controller and soon also in spacebindingcleanup_controller (for the SBR cleanup task) 🤔 

Thanks for your suggestion, I'll address it in a separate PR 👍